Dockerfile-alpine.template

```diff
@@ -1,29 +1,48 @@
 FROM alpine:0.0
 
+ARG CHECKSUM=
+
 ENV NODE_VERSION 0.0.0
 
-RUN addgroup -g 1000 node \
-    && adduser -u 1000 -G node -s /bin/sh -D node \
-    && apk add --no-cache \
-        libstdc++ \
-    && apk add --no-cache --virtual .build-deps \
-        binutils-gold \
-        curl \
-        g++ \
-        gcc \
-        gnupg \
-        libgcc \
-        linux-headers \
-        make \
-        python \
-  # gpg keys listed at https://github.com/nodejs/node#release-keys
-  && for key in \
-    "${NODE_KEYS[@]}"
-  ; do \
-    gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
-    gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
-    gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
-  done \
+RUN ARCH= && alpineArch="$(arch)" \
+  && case "${alpineArch##*-}" in \
```

Comment:
This indent is different with the origin one, I don't think we should touch it here.